docs/section22/Rule22-5.md

```diff
@@ -0,0 +1,59 @@
+
+# CHW&CW - Rule 22-5  
+
+**Rule ID:** 22-5  
+**Rule Description:** For Baseline chilled water loop that is not purchased chilled water and serves computer room HVAC systems (System Type-11), chilled-water supply temperature shall be reset higher based on the HVAC system requiring the most cooling.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.9 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system serves computer room HVAC system (System Type-11)
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water.
+3. Pass Rule 22-3.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system serves computer room HVAC system (System Type-11):
+ `LEN(hvac.does_serve_computer_room == TRUE for hvac in B-RMR...heating_ventilation_air_conditioning_systems) > 0`
+
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water: `if B_RMR.ASHRAE229.chillers:`
+
+3. Pass Rule 22-3: `if Rule-22-3 == "PASS"`
+
+## Rule Logic:  
+
+- For each HVAC system in B-RMR: `for hvac_b in B-RMR...heating_ventilation_air_conditioning_systems:`
+
+  - Check if HVAC system serves computer room: `if hvac_b.does_serve_computer_room:`
+
+    - Save CHW loop that serves the cooling system of HVAC system serving computer room to an array: `chw_loop_for_computer_room_array.append(hvac_b.cooling_system.chilled_water_loop)`
+
+- For each chiller in B_RMR: `for chiller_b in B_RMR.ASHRAE229.chillers:`
+
+  - Get cooling loop that chiller servers: `chw_loop_b = chiller_b.cooling_loop`
+
+    - If cooling loop has not been saved in CHW loop array and serves computer room HVAC system: `if ( chw_loop_b NOT in chw_loop_reset_check_array ) AND ( chw_loop_b in chw_loop_for_computer_room_array ):`
+
```

Comment:
This would need to be updated per above, to remove computer room criteria

Reply:
Not sure why we need to remove computer room criteria here. Is it because we assume there can only be one primary CHW loop in Baseline (so that the computer room system can only be served by this loop)? If so, to summarize the logic:
1). check baseline does not use purchased CHW, as part of applicability check
2). check if there is Sys-11 in baseline, as part of applicability check
3). if so, get CHW primary loop and check if temperature reset is "LOAD_RESET" type

Reply:
i believe your approach and the checks you have summarized are correct.